### PR TITLE
Reworked "Orphaned or Unowned Files"

### PR DESCRIPTION
**Pull request by jsegitz**

### CODE REVIEW

File: xml/hardening.xml [hunk 1]
```diff
@@ -3075,7 +3075,35 @@ drwxrwxrwt 18 root root 16384 Dec 23 22:20 /tmp</screen>
...
+    You might need to extend the list of directories that are searched if you have a different filesystem structure.
+    </para>
+
+    <para>
+     A different problem are files that were not installed via the packaging
+     system and therefor don't receive updates. You can check for such files with
+     the following command:
+    </para>
+
+<screen># find /bin /lib /lib64 /usr -path /usr/local -prune -o -type f -a -exec /bin/sh -c "rpm -qf {} &amp;> /dev/null || echo {}" \;</screen>
```
Comment (svenseeberg) [inline xml/hardening.xml]: replace number sign (#) with \&root.prompt;

Comment (svenseeberg) [inline xml/hardening.xml]: Replace number sign (#) with \&prompt.root;

File: xml/hardening.xml [hunk 2]
```diff
@@ -3075,7 +3075,35 @@ drwxrwxrwt 18 root root 16384 Dec 23 22:20 /tmp</screen>
...
+
+    <para>
+     You should run this command as an untrusted user (e.g. nobody) since crafted filenames might lead to command
```
Comment (svenseeberg) [inline xml/hardening.xml]: Remove "You should". "Run this command ..." is fine. "e.g." -> "for example"

File: xml/hardening.xml (hunk 1 above)
Comment (jsegitz) [reply xml/hardening.xml]: This command must not be run as root

File: xml/hardening.xml (hunk 2 above)
Comment (jsegitz) [reply xml/hardening.xml]: Thanks

File: xml/hardening.xml (hunk 1 above)
Comment (svenseeberg) [reply xml/hardening.xml]: oh okay, then it should be > for which we have the \&prompt.user; entity. I'll fix it after the merge.

**Pull request merged**